### DISCUSSION TITLE
Remote setting wiped out

### DISCUSSION THREAD

**Discussion by tatsuo777github** [GerryDazoo/Slinger#636] [General]
Hi Gerry

This is related to the discussion "Slinger M1 Custom Remote lost after reboot" #593. 
I have new M1s and M2s and tried to set them up however, the remote setting (I am using Panasonic PVR) is not retained after configuring slingbox once the slingbox is powerd off. I was thinking to use UPS to backup slingbix but today I found the remote setting is gone even without powering the unit off.  The video streaming always works fine so I am starting to consider to look for IR over IP device for remote operation.
May I ask if you are aware of this and have any workaround ?

Tatsuo

**Comment by GerryDazoo** [comment]
You can try loading a compatible custom remote configuration. 
http://www.hifi-remote.com/wiki/index.php/How_to_Install_a_Custom_Remote_onto_the_Slingbox

**Comment by tatsuo777github** [comment]
Hi Gerry
Thanks for your reply. I used custom .bin file. 
After testing remote and clicked "yes the remote works" , the configuring clock symbol never end up spinning. It seems the process is not completed. When I terminate the Desktop app and go to Slingbox Directly screen, the device is not   "connect"  but still in "setup". 
The remote setup is evaporated when the device is powered off or after a few days even if not powered off. This symptom is consistent among all of my  M1/M2s.
I guess this has to do with Slingbox server that is somewhat working? I am afraid if they completely remove the server, we cannot setup new devices at all? Do we have a solution that allows us to setup the device off line?

Tatsuo
